TravelAgency-spring-mvc/src/main/webapp/WEB-INF/jsp/customer/registration.jsp

```diff
@@ -17,7 +17,6 @@
             <form:form method="post" action="${pageContext.request.contextPath}/customer/registration"
```

Comment:
redirect to /create instead of /registration - your controllers are set up that way